src/Engines/nativeEngine.ts

```diff
@@ -95,6 +95,8 @@ interface INativeEngine {
     clearDepth(depth: number): void;
     clearStencil(stencil: number): void;
 
+    exit(exitCode: number): void;

```

Comment:
We discussed this and this will be removed and moved into the TestApp in BabylonNative.